axiom/tests/test_txn.c

```diff
@@ -8068,6 +8084,256 @@ static void test_segment_record_error(void) {
   nr_txn_destroy(&txn);
 }
 
+static nrtxn_t* new_txn_for_record_log_event_test(char* entity_name) {
+  nrapp_t app;
+  nrtxnopt_t opts;
+  nrtxn_t* txn;
+  nr_segment_t* segment;
+
+  /* Setup app state */
+  nr_memset(&app, 0, sizeof(app));
+  app.state = NR_APP_OK;
+  app.entity_name = entity_name;
+
+  /* Setup log feature options */
+  nr_memset(&opts, 0, sizeof(opts));
+  opts.logging_enabled = true;
+  opts.log_forwarding_enabled = true;
+  opts.log_events_max_samples_stored = true;
```

Comment:
This takes an integer value doesn't it?

Reply:
Addressed in 6683408e0823c8746b73219a130f9cd9e56edcd2